src/lib/arch/aarch64/Architecture.cc

```diff
@@ -297,36 +296,50 @@ void Architecture::updateSystemTimerRegisters(RegisterFileSet* regFile,
 }
 
 std::vector<RegisterFileStructure>
-Architecture::getConfigPhysicalRegisterStructure(YAML::Node config) const {
+Architecture::getConfigPhysicalRegisterStructure() const {
+  // Get register counts
+  ryml::ConstNodeRef regConfig = config::SimInfo::getConfig()["Register-Set"];
+  uint16_t gpCnt;
```

Comment:
Use of `Count` rather than `Cnt` is preferable in my opinion. Won't die on this hill though so change at your discression

Reply:
That's fine by me given prior changes in comments further up